src/sass/partials/layouts/_header.scss

```diff
@@ -1,17 +1,22 @@
-.header {
-  background-color: color('gray', 90);
+._header {
+  background-color: setting-get('black');
   height: 75px;
+  margin-top: 0;
 }
 
-.nav {
-
-  @include vertical-center(50%, false);
+.header {
```

Comment:
Here, `.header` w/o a leading `_` is a component, not a layout. Please move it (and related styling) to `components`.

Also, is there something more descriptive? Having `_header` and _header` may lead to confusion. `site-header` maybe?